test/fixtures/call_expressions.json

```diff
@@ -213,7 +213,7 @@
         {
             "type": "Junk",
             "annotations": [],
-            "content": "shuffled-args = {FUN(1, x: 1, \"a\", y: \"Y\", msg)}\n"
+            "content": "shuffled-args = {FUN(1, x: 1, \"a\", y: \"Y\", msg)}\n\n"
```

Comment:
Great question! I'm not happy with the answer, but the good news is that this PR will help in making this right.

The reference fixtures in `fluent-syntax` are copied directly from the reference parser's tests. They're not generated by the tooling parser like the other kinds of fixtures. I've taken a note to document this in a `README`.

So what you're seeing in `fluent-syntax/test/fixtures_reference` are actually the [reference parser's fixtures from Syntax 0.7](https://github.com/projectfluent/fluent/blob/v0.7.0/test/fixtures/call_expressions.json#L216).

The actual output of the tooling parser includes those trailing blank lines. You can verify that in the Playground. The `fixtures_reference` tests pass in `fluent-syntax` because the [test runner explicitly ignores `Junk`](https://github.com/projectfluent/fluent.js/blob/a6bab32256e0148aa3256ba949cde1b583be0fe3/fluent-syntax/test/reference_test.js#L49-L52) due to its being parsed differently in Syntax 0.7. With this PR, we're getting much closer to being able to test junk too :)